fuzzing/fuzzer.go

```diff
@@ -803,12 +807,15 @@ func (f *Fuzzer) Start() error {
 		return err
 	}
 
+	// Measure reversion stats across all workers
+	cleanup := f.ReversionStats.StartWorker(f.ctx)
+
```

Comment:
```suggestion
	cancel := f.ReversionStats.StartWorker(f.ctx)
	defer cancel

```
Is there any reason to call cleanup where it is or it can be deferred?

Reply:
I'll find a way to move the cleanup responsibilities elsewhere. the caller probably shouldn't have to worry about this anyways